fredericoo/saltplay
# feat(perf): revert pages to static and implement on-demand ISR

# Overview
This PR aims to increase our performance metrics by making pages all the pages static again, but with support for the admin dashboard to trigger their revalidation. 

## What we have done

- Made every page static again
- Removed Loading bar at the top of pages (they're instant again)
- Installed bundle analyser to visualise our bundle size and act upon it
- Dynamically importing modal contents (for new match, and feedback)
- Added cache for 1 minute for every page, 10 minutes for the home page
- Added revalidation to pages changed in every API endpoint for the dashboard

## How to test

- Run preview and make edits to games, players, and offices. They should take 1 minute on regular pages to be refreshed, and 10 minutes on the home page.

## Checklist:

- I have created automated tests to replicate my manual testing automatically
